From e3ae486cbe718f8fef45d875901e1f11e25306d0 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <tom.zanussi@intel.com>
Date: Tue, 3 Jul 2012 13:07:21 -0500
Subject: [PATCH] perf: use pkg-config instead of python-config

Python has build flags available via pkg-config, use those at
build-time instead.

Signed-off-by: Tom Zanussi <tom.zanussi@intel.com>
---
 tools/perf/Makefile.config | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/perf/Makefile.config b/tools/perf/Makefile.config
index 63f534a0902f..ef7597af8dd0 100644
--- a/tools/perf/Makefile.config
+++ b/tools/perf/Makefile.config
@@ -180,10 +180,10 @@ strip-libs  = $(filter-out -l%,$(1))
 PYTHON_CONFIG_SQ := $(call shell-sq,$(PYTHON_CONFIG))
 
 ifdef PYTHON_CONFIG
-  PYTHON_EMBED_LDOPTS := $(shell $(PYTHON_CONFIG_SQ) --ldflags 2>/dev/null)
+  PYTHON_EMBED_LDOPTS := $(shell pkg-config --libs python 2>/dev/null)
   PYTHON_EMBED_LDFLAGS := $(call strip-libs,$(PYTHON_EMBED_LDOPTS))
   PYTHON_EMBED_LIBADD := $(call grep-libs,$(PYTHON_EMBED_LDOPTS)) -lutil
-  PYTHON_EMBED_CCOPTS := $(shell $(PYTHON_CONFIG_SQ) --cflags 2>/dev/null)
+  PYTHON_EMBED_CCOPTS := $(shell pkg-config --cflags python 2>/dev/null)
   ifeq ($(CC_NO_CLANG), 1)
     PYTHON_EMBED_CCOPTS := $(filter-out -specs=%,$(PYTHON_EMBED_CCOPTS))
   endif
-- 
2.5.0

